cmd/user-agent/src/pages/TablePresentation.vue

```diff
@@ -153,7 +153,7 @@ SPDX-License-Identifier: Apache-2.0
             privateKey: privateKey,
             keyType: keyType
           }).then(resp => {
-                   this.vpData = JSON.stringify(resp.verifiablePresentation)
+            this.vpData = JSON.parse(JSON.stringify(resp.verifiablePresentation))
```

Comment:
is this working

Reply:
set resp.VerifiablePresentation :)
convertng to string and converting to JSON again :)